ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client.hs

```diff
@@ -228,11 +230,11 @@ newtype Our a = Our { unOur :: a }
 -- data from 'ChainSyncState'.
 viewChainSyncState ::
   IOLike m =>
-  StrictTVar m (Map peer (ChainSyncClientHandle m blk)) ->
+  STM m (Map peer (ChainSyncClientHandle m blk)) ->
   (ChainSyncState blk -> a) ->
   STM m (Map peer a)
 viewChainSyncState varHandles f =
-  Map.map f <$> (traverse (readTVar . cschState) =<< readTVar varHandles)
+  Map.map f <$> (traverse (readTVar . cschState) =<< varHandles)
```

Comment:
Rename `varHandles` into `readHandles` for consistency with other places?

Reply:
Sounds good.

Reply:
Done in 7259d9254e6d978fded9edd3390e1925f08a7d66.